help/markdown/fake-tools-octo.md

```diff
@@ -19,6 +19,13 @@ You will also need to install and configure at least one [Tentacle](http://octop
 
 This module is a wrapper around the [Octo.exe](https://octopus.com/docs/api-and-integration/octo.exe-command-line) CLI tool which controls Octopus Deploy API. You'll need the Octo.exe tool itself accessible to your FAKE script. Download it from [here](https://octopus.com/downloads).
 
+This module also supports use via a .NET Core Tool-Manifest. Installation is simple! From the root of your repository run the following
+
+```bash
+dotnet new tool-manifest # if one doesn't already exist
+dotnet tool install Octopus.DotNet.Cli 
+```
```

Comment:
Shouldn't we add here that you need to set the ToolPath and maybe link to https://fake.build/dotnet-cli.html#SDK-tools-local-global-clireference?